mender-convert-modify

```diff
@@ -219,11 +219,4 @@ platform_modify
 for overlay in "${overlays[@]}"; do
   log_info "Applying rootfs overlay: ${overlay}"
   run_and_log_cmd "sudo rsync --archive --keep-dirlinks --verbose ${overlay}/ work/rootfs/"
-
-  for append in $(cd work/rootfs; find . -name '*.mender-convert-append'); do
-      sourcefile=$(echo $append | sed 's@.mender-convert-append$@@')
-      log_info "Applying rootfs overlay append to: ${sourcefile}"
-      run_and_log_cmd "sudo cat work/rootfs/${append} >> work/rootfs/${sourcefile}"
-      run_and_log_cmd "sudo rm -f work/rootfs/${append}"
-  done
```

Comment:
If the intention is to drop this, I suggest to drop the first commit instead of removing it here.

Reply:
Yeah, I forgot I had done that as a separate commit. Fixed.